pyproject.toml

```diff
@@ -62,9 +62,6 @@ testpaths = [
     "tests",
 ]
 
-[tool.ruff]
-target-version = "py310"
```

Comment:
Looking at the docs for this, ruff will look at `requires-python` and actually recommends just setting that instead of this.